src/Symfony/Component/VarExporter/ProxyHelper.php

```diff
@@ -86,7 +86,7 @@ public static function generateLazyGhost(\ReflectionClass $class): string
                 .($p->isProtected() ? 'protected' : 'public')
                 .($p->isProtectedSet() ? ' protected(set)' : '')
                 ." {$type} \${$name}"
-                .($p->hasDefaultValue() ? ' = '.$p->getDefaultValue() : '')
+                .($p->hasDefaultValue() ? ' = '.VarExporter::export($p->getDefaultValue()) : '')
```

Comment:
actually this should use the exportDefault helper
and the PR should target branch 6.4